styles.css

```diff
@@ -0,0 +1,134 @@
+* {
+  padding: 0;
+  margin: 0;
+}
+
+header {
+  margin: 10px;
+  padding: 10px;
+}
+
+.nav-menu-content {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.nav-menu {
+  list-style: none;
+  display: flex;
+  gap: 60px;
+}
+
+a:hover {
+  background: #acb7c3;
+  transition: 0.4s;
+}
+
+.logo {
+  font-size: 18px;
+  line-height: 20px;
+  font-weight: bold;
+  color: #6070ff;
+}
+
+.hamburger {
+  display: none;
+  cursor: pointer;
+}
+
+.bar {
+  display: block;
+  width: 25px;
+  height: 3px;
+  margin: 5px auto;
+  -webkit-transition: all 0.3s ease-in-out;
+  transition: all 0.3s ease-in-out;
+  background-color: #6070ff;
+}
+
+@media (max-width: 768px) {
+  .hamburger {
+    display: block;
+  }
+
+  .hamburger.active .bar:nth-child(2) {
+    opacity: 0;
+  }
+
+  .hamburger.active .bar:nth-child(1) {
+    transform: translateY(8px) rotate(45deg);
+  }
+
+  .hamburger.active .bar:nth-child(3) {
+    transform: translateY(-8px) rotate(-45deg);
+  }
+
+  .nav-menu {
+    position: fixed;
+    left: -100%;
+    top: 70px;
+    gap: 0;
+    flex-direction: column;
+    background-color: #6070ff;
+    width: 100%;
+    text-align: center;
+    transition: 0.3s;
+  }
+
+  .nav-items {
+    margin: 16px 0;
+  }
+
+  .nav-menu.active {
+    left: 0;
+  }
+}
+
+.homepage {
+  background-image: url("image/backimg.png");
+  background-repeat: no-repeat;
+  background-size: cover;
+  margin: auto;
+  overflow: hidden;
+  min-height: 600px;
+}
```

Comment:
- [ ] Kindly implement requested changes on the previous [pull request open
](https://github.com/tasnimulhasan07/scel2/pull/1)   the previous reviewer requested you make the background image look like the Figma design, check screenshot below ⬇️

Figma design           |  Your design
:-------------------------:|:-------------------------:
![designB](https://user-images.githubusercontent.com/69638013/153697309-d7e9176c-02be-43da-bb80-e7f1f3050df4.PNG) | ![bgimg](https://user-images.githubusercontent.com/69638013/153697490-afd22204-6853-42a3-ae21-a2438553d772.PNG)

Reply:
br has been removed and padding and margin has been added as per request